tests/flow/test_variable_length_traversals.py

```diff
@@ -182,3 +182,19 @@ def test10_filtered_edges_after_segment_change(self):
         actual_result = redis_graph.query(query)
         expected_result = [['A', 'B']]
         self.env.assertEquals(actual_result.result_set, expected_result)
+
+    # Test variable-length edges using minHops and maxHops
+    def test11_variable_length_edges(self):
+        query = """CREATE (a {v:'a'}), (b {v:'b'}), (c {v:'c'}), (d {v:'d'}), (a)-[:R]->(b), (b)-[:R]->(c), (c)-[:R]->(a), (d)-[:R]->(d)"""
+        actual_result = redis_graph.query(query)
+        query_to_expected_result = {
+            "MATCH p = ({v:'a'})-[*2]-({v:'c'}) RETURN length(p)" : [[2]],
```

Comment:
this one is not variable length it fixed length

Reply:
You are right, it's fixed length.
Should I rename the ```QGEdge_VariableLength()``` method and the property ```varLen```? Because with the change of this PR, it detects if a range length was specified, fixed or variable.

And maybe the function ```NewCondVarLenTraverseOp()``` should be renamed too.

Reply:
I renamed some functions. Please check if you agree.
```QGEdge_RangeLength()``` to ```QGEdge_RangeLength()```
```_AlgebraicExpression_IsVarLen()``` to ```_AlgebraicExpression_IsRangeLen()```

Property ```varLen``` renamed to ```rangeLen```

The test were not removed, just renamed, for all tests  ```rangeLen == true```
```test11_variable_length_edges()``` to ```test11_range_length_edges()```